cfgov/jinja2/v1/_includes/atoms/large_checkbox.html

```diff
@@ -0,0 +1,57 @@
+{# ==========================================================================
+
+   large_checkbox.render()
+
+   ==========================================================================
+
+   Description: Builds checkboxes fields with a large hit target.
+
+   settings (required):
+
+   settings.legend:     Name of the field.
+
+   settings.fields:     An array of all the checkbox labels.
+
+   settings.size:       The size of the field. Defaults to 1-2.
+                        Possible options: 1, 1-2, 1-4
```

Comment:
Hey, so far, filters haven't been using the big checkboxes/radio targets. I think it's better to reserve those form forms because of the additional real estate they require. Filters we often want to make as concise as possible.